packages/calypso-babel-config/presets/default.js

```diff
@@ -27,7 +27,13 @@ module.exports = ( api, opts ) => ( {
 		],
 		[
 			require.resolve( '@babel/preset-react' ),
-			{ runtime: 'automatic', importSource: '@emotion/react' },
+			{
+				runtime: 'automatic',
+				importSource:
+					typeof process.env.IMPORT_SOURCE !== 'undefined'
+						? process.env.IMPORT_SOURCE
+						: '@emotion/react',
```

Comment:
I realize `@emotion/react` isn't a specified dependency of the package. Should it be added specifically if we're relying on it like this?

Reply:
Not sure if it makes sense to add it as a dependency of `calypso-babel-config` as it only rewrites the imports and doesn't seem to add anything additional to the build.

However, it should specified for each package that uses it, as it's currently missing in some.